packages/cli/src/license/license.controller.ts

```diff
@@ -14,7 +16,18 @@ export class LicenseController {
 	@Post('/enterprise/request_trial')
 	@GlobalScope('license:manage')
 	async requestEnterpriseTrial(req: AuthenticatedRequest) {
-		await this.licenseService.requestEnterpriseTrial(req.user);
+		try {
+			await this.licenseService.requestEnterpriseTrial(req.user);
+		} catch (error: unknown) {
+			if (error instanceof Error) {
+				const errorMsg =
+					(error as AxiosError<{ message: string }>).response?.data?.message ?? error.message;
+
+				throw new BadRequestError(errorMsg);
```

Comment:
This is relying on the error message that the license server returns. Maybe we should update the license server to return error codes, so that we can map those to human readable errors in this repo, instead of having to update the license server every time product wants to change the error that gets displayed.

Reply:
> This is relying on the error message that the license server returns.

This is in a workflow, but I agree with the general principle.